# DP: Description: support alternative gettext tree in
# DP: /usr/share/locale-langpack
# DP: Patch author: Jeff Bailey <jeff.bailey@canonical.com>
# DP: Upstream status: Ubuntu-Specific
# DP: Date: 2006-06-06 11:46:57 EDT

--- glibc-2.4/locale/findlocale.c.old	2006-06-06 11:42:29.000000000 -0400
+++ glibc-2.4/locale/findlocale.c	2006-06-06 11:43:22.000000000 -0400
@@ -56,7 +56,7 @@ struct locale_data *const _nl_C[] attrib
    which are somehow addressed.  */
 struct loaded_l10nfile *_nl_locale_file_list[__LC_LAST];
 
-const char _nl_default_locale_path[] attribute_hidden = LOCALEDIR;
+const char _nl_default_locale_path[] attribute_hidden = LOCALEDIR ":/usr/share/locale-langpack";
 
 
 struct locale_data *
